### PR TITLE
Fix world-map to avoid set multiple listener on the ws

### PR DESCRIPTION
**Pull request by simoexpo**

### CODE REVIEW

File: src/components/world-map.vue [hunk 1]
```diff
@@ -23,6 +23,7 @@ export default class extends Vue {
 
   private map: any | MapEngine = undefined;
   private socketURL: string = 'dvs';
+  private listener: any | Promise<void> = undefined;
```
Comment (massimosiani) [inline src/components/world-map.vue]: `any | <other type>` is equivalent to `any`. If you want to express nullability, you may use
`private listener?: Promise<void> = undefined;`

File: src/components/world-map.vue [hunk 2]
```diff
@@ -83,17 +84,19 @@ export default class extends Vue {
...
-      socket
-        .pipe(
-          filter((event: any) =>
-            event.eventType === 'FlightList',
-          ),
-          map((event: any) => this.manageFlightList(event.eventPayload)),
-        )
-        .subscribe();
-    });
+    if (!this.listener) {
```
Comment (massimosiani) [inline src/components/world-map.vue]: is there a way to encode idempotency in the type instead?

File: src/components/world-map.vue (hunk 1 above)
Comment (simoexpo) [reply src/components/world-map.vue]: should we also chage `private map: any | MapEngine = undefined;`?

Comment (massimosiani) [reply src/components/world-map.vue]: to be honest, yes

**Pull request merged**